src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/Public.Data/IDataSetService.cs

```diff
@@ -16,4 +16,10 @@ public interface IDataSetService
         int pageSize,
         Guid publicationId,
         CancellationToken cancellationToken);
+
+    Task<Either<ActionResult, DataSetVersionSummaryViewModel>> GetVersion(
+        Guid releaseVersionId,
+        Guid dataSetId,
+        string dataSetVersion,
+        CancellationToken cancellationToken = default);
```

Comment:
That's true. We have been using `default` for all the public API stuff, but I do agree it's not really necessary. I suppose it means we don't have to specify one in unit tests though (if we had some)?

Reply:
I only raised this because @ntsim left me a similar comment about `IDataSetService.ListPublicationDataSets` which is also only called from the controller and has no unit tests, only integration tests.

I don't mind using `default` but we should be consistent about it. One for us to get together and discuss?